jujugui/static/gui/src/app/assets/css/_mixins.scss

```diff
@@ -262,7 +262,8 @@
     color: $cool-grey;
     overflow-x: hidden;
     overflow-y: auto;
-    .spinner_loading {
```

Comment:
Ah yeah, no worries. FYI we use the double underscore, double hyphen style: https://en.bem.info/method/naming-convention/#alternative-naming-schemes